tools/lib/sanity_check.py

```diff
@@ -6,10 +6,10 @@
 def check_venv(filename: str) -> None:
     try:
         import django
-        import ujson
+        import orjson
         import zulip
         django
-        ujson
+        orjson
```

Comment:
We don't want to replace this with `orjson`; the intent of this check is to look for a module likely to be in any Zulip venv, as it's used to print something about `PROVISION_VERSION`.  I switched to `bitfield` which has been a dependency of Zulip since 2013 and is probably not going away soon.

We'll see the failure mode of Zulip printing: "You need to run test-backend inside a Zulip dev environment" when folks rebase back in time and fail to provision after this commit.  Oh well.